src/helper-functions/fetch.js

```diff
@@ -30,7 +30,6 @@ const fetch = async (
     });
     return response;
   } catch (err) {
-    console.error('Something went wrong. Please contact admin', err);
```

Comment:
Why remove this line ?? I guess you can use alert here if this is causing es-lint errors or you can check for the rules to follow
https://docs.w3cub.com/eslint/rules/no-console

Reply:
Please add this back and set the rule `'no-console' : 'off'`

Reply:
@rohan09-raj , found an exception on the rule **no-console**: https://eslint.org/docs/rules/no-console. Should I apply this instead of disabling the entire rule?

Reply:
Yes, I think we can use this.